### ISSUE TITLE
NJ bus and train data via fetch_feeds are out-of-date

### ISSUE THREAD

**Issue by ktohalloran** [azavea/gtfs-feed-fetcher#14]
For the last 2 deploys (July & August), the NJ bus and train data downloaded by `fetch_feeds.py` was not the most current data and did not match the data available through the NJ Transit developer portal. We should update `fetch_feeds` to download the data from the portal; unfortunately, it looks like this might involve reverting a part of the fix in [PR 13](https://github.com/azavea/gtfs-feed-fetcher/pull/13), though maybe we can see how Transitland is handling this issue and update it a similar way.

**Comment by lukemckinstry** [comment]
I and @ktohalloran were unable to reproduce this. I downloaded files via 
1) this repo  
2) transitland
3) the [NJTransit dev portal](https://www.njtransit.com/users/developer-tools) 

and they were identical. So it appears whatever issue was occurring has been fixed. And the good news is that the Results table produced by this script caught that the files downloaded were out of date, so we should continue to rely on that.

As to why the error may have occured, it appears the NJTransit dev portal uses different URLs from this repo & transitland, so it's possible there was human error and NJTransit let them get out of sync.
